draft-ietf-tls-external-psk-importer.md

```diff
@@ -195,7 +203,33 @@ PSKs for TLS 1.2. Indeed, this is necessary for incremental deployment.
 
 # Security Considerations
 
-DISCLAIMER: This is a WIP draft and has not yet seen significant security analysis.
+The Key Importer security goals can be roughly stated as follows: avoid PSK re-use across
```

Comment:
Is there a distinction to be made between goals of the protocol and the properties we claim it achieves?
I would say preventing IKM reuse is a goal, but not a property. The property is key separation by KDF. 
Maybe this is unnecessarily specific.

Reply:
It's probably best to keep them separate, yeah. Good suggestion! I'll try and clear this up.